sklearn/metrics/_dist_metrics.pxd.tp

```diff
@@ -78,27 +78,82 @@ cdef class DistanceMetric{{name_suffix}}:
     # we must define them here so that cython's limited polymorphism will work.
     # Because we don't expect to instantiate a lot of these objects, the
     # extra memory overhead of this setup should not be an issue.
-    cdef {{DTYPE_t}} p
-    cdef {{DTYPE_t}}[::1] vec
-    cdef {{DTYPE_t}}[:, ::1] mat
+    cdef DTYPE_t p
+    cdef DTYPE_t[::1] vec
+    cdef DTYPE_t[:, ::1] mat
     cdef ITYPE_t size
     cdef object func
     cdef object kwargs
 
-    cdef DTYPE_t dist(self, const {{DTYPE_t}}* x1, const {{DTYPE_t}}* x2,
-                      ITYPE_t size) nogil except -1
-
-    cdef DTYPE_t rdist(self, const {{DTYPE_t}}* x1, const {{DTYPE_t}}* x2,
-                       ITYPE_t size) nogil except -1
-
-    cdef int pdist(self, const {{DTYPE_t}}[:, ::1] X, {{DTYPE_t}}[:, ::1] D) except -1
-
-    cdef int cdist(self, const {{DTYPE_t}}[:, ::1] X, const {{DTYPE_t}}[:, ::1] Y,
-                   {{DTYPE_t}}[:, ::1] D) except -1
-
-    cdef DTYPE_t _rdist_to_dist(self, {{DTYPE_t}} rdist) nogil except -1
-
-    cdef DTYPE_t _dist_to_rdist(self, {{DTYPE_t}} dist) nogil except -1
+    cdef DTYPE_t dist(self,
+        const {{INPUT_DTYPE_t}}* x1,
+        const {{INPUT_DTYPE_t}}* x2,
+        ITYPE_t size,
+    ) nogil except -1
```

Comment:
```suggestion
    cdef DTYPE_t dist(
        self,
        const {{INPUT_DTYPE_t}}* x1,
        const {{INPUT_DTYPE_t}}* x2,
        ITYPE_t size,
    ) nogil except -1
```
This would be more like black.